BlazarUI/app/scripts/components/shared/SearchFilter.jsx

```diff
@@ -40,14 +42,23 @@ class SearchFilter extends Component {
     this.refs.searchFilterInput.focus();
   }
 
-  handleChange() {    
+  handleChange() {
     this.setState({
       searchValue: this.refs.searchFilterInput.value
     });
     this.handleSearchDebounced();
   }
 
-  render() {    
+  handleKeyup(e) {
+    const notFocusedOnInputElement = e.target === document.body || e.target.tagName === 'A';
+    const modifierKey = e.metaKey || e.shiftKey || event.ctrlKey;
+    const shortcutPressed = FOCUS_SEARCH_BAR_SHORTCUTS.includes(e.which);
```

Comment:
I've been using underscore functions for the sake of consistency, but I don't think it matters much either way (hopefully none of our devs are using IE 🙊)

Maybe just use `contains` to match usage in the rest of the app  -we can swap to `includes` in the future if we want to